apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-content-palette/dot-content-palette.component.scss

```diff
@@ -36,9 +49,12 @@
 .dot-content-palette__items {
 
     overflow: hidden auto;
+    display: flex;
+    flex-wrap: wrap;
+    gap: $spacing-1;
 
     div {
-        width: 7.2rem;
+        width: 48%;
```

Comment:
This math will not add... I think we can do css grid here and aspect ratio css because this will not add it.